pycosep/community_separability.py

```diff
@@ -541,9 +543,8 @@ def compute_separability(embedding, communities, positives=None, variant=Separab
             metadata[index_group_combination]["best_tour"] = best_tour
 
             if not best_tour.size == 0:
-                separability_path = _convert_tour_to_one_dimension(best_tour, pairwise_data, pairwise_communities)
-                scores, source_nodes, target_nodes, edge_weights, cut_start_node, cut_end_node = separability_path
-
+                scores, source_nodes, target_nodes, edge_weights, cut_start_node, cut_end_node = _convert_tour_to_one_dimension(
```

Comment:
Hmm, this and others are changes from the previous version. Please merge the latest `main` into yours so you do not have to carry this over.

Reply:
Yes. This was caused by editing the code from an outdated main. I have changed to the latest one.